files/en-us/web/javascript/reference/global_objects/date/setdate/index.html

```diff
@@ -50,11 +50,11 @@ <h2 id="Examples">Examples</h2>
 
 <h3 id="Using_setDate">Using setDate()</h3>
 
-<pre class="brush: js">var theBigDay = new Date(1962, 6, 7); // 1962-07-07 (7th of July 1962)
+<pre class="brush: js">var theBigDay = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962)
```

Comment:
Could you update this to use `const`?
```suggestion
<pre class="brush: js">const theBigDay = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962)
```